content/post/benchmarking-gnns/index.md

```diff
@@ -0,0 +1,249 @@
+
+>Importantly, this strategy makes it easy for the users to not fathom on choosing how many epochs to train their model for.
+
+ Each experiment is run on $4$ different seeds for a maximum of $12$ hours of training time and the summary statistics of the last epoch score of the $4$ experiments is reported.
+
+**Parameter budget:** We decide on using two trainable parameter budgets: (i) $100k$ parameters for each GNNs for all the tasks, and (ii) $500k$ parameters for GNNs for which we investigate scaling a model to larger parameters and deeper layers. The number of hidden layers and hidden dimensions are selected accordingly to match these budgets. 
+
+We make this choice of having a similar parameter budget for fair comparison because it becomes otherwise difficult to rigorously evaluate different models. In GNN literature, it is often seen the a new model is compared to the existing literature without any detail of the number of parameters, or any attempt to have the same size of the model. Having said that, our goal is not to find the optimal set of hyperparameters for each of the models which is a compute-intensive task.
+
+---
+
+#### d. Graph Neural Networks:
+We benchmark two broad classes of GNNs that represent the categorical advances in the architectures of a graph neural network witnessed in the most recent literature. We call the two classes as **GCNs (Graph Convolutional Networks)** and **WL-GNNs (Weisfeiler-Lehman GNNs)**.
+
+>GCNs refer to the popular message-passing based GNNs which leverage sparse tensor computation and WL-GNNs are the theoretically expressive GNNs based on the WL-test to distinguish non-isomorphic graphs which require dense tensor computation at each layer. 
+
+Accordingly, our experimental pipeline is shown in Fig 5 for GCNs and Fig 6 for WL-GNNs.
+
+{{< figure src="mpgcns.png" title="Fig 5: Our standard experimental pipeline for GCNs which operate on *sparse* rank-$2$ tensors." lightbox="true" width="100%">}}
+{{< figure src="wlgnns.png" title="Fig 6: Our standard experimental pipeline for WL-GNNs which operate on *dense* rank-$2$ tensors." lightbox="true" width="100%">}}
+
+We direct the readers to our paper and the corresponding works for more details on the mathematical formulations of the GNNs. 
+
+---
+---
+
+> _For a quick recap at this stage, we discussed the **need of a benchmark**, the **challenges** in building such a framework and **details on our proposed benchmarking framework**. We now delve into the experiments._
+
+We perform a principled investigation into the message passing based GCNs and the WL-GNNs to reveal important insights and highlight critical underlying challenges in building a powerful GNN model.
+
+---
+
+### Benchmarking GNNs on the proposed datasets.
```

Comment:
This section seems to have some rendering issues for latex math (at least on my Chrome browser via Netlify). Are you able to see the math equations render correctly for you, locally?

Reply:
I also do not see the math eqns rendered. In fact, when I run your transformer post locally, I do not see rendered latex eqns too.

I presumed that it would be reflected in the published post, like it does for the transformer blog. But idk how it would work.